### ISSUE TITLE
FingerTip Dataset Format

### ISSUE THREAD

**Issue by Lisandro79** [tucan9389/FingertipEstimation-CoreML#2]
Hi @tucan9389 ,

I am working on a project that requires finger tracking. I hope you don't mind me asking you a couple of basic questions. I would like to use only 1 keypoint ("FingerTip") during training, and I am using the annotation tool from "don't be turtle" project to create my own dataset. 

Did you use "background" images in your training set? I mean, did you have images without any index finger in them (just background)?

If so, would you mind sharing with me one example of how the keypoints were annotated in your training set (i.e., what values you set in the keypoints lists)? 

Thanks a lot

Cheers

**Comment by tucan9389** [comment]
Hi @Lisandro79 

> Did you use "background" images in your training set? I mean, did you have images without any index finger in them (just background)?

No. I took videos using my iPhone and split to images. The split images contain various background. And I just annotate only 1 keypoint (Fingertip. The endpoint of the index finger) for all split images.

> If so, would you mind sharing with me one example of how the keypoints were annotated in your training set (i.e., what values you set in the keypoints lists)?

When I find the fingertip dataset, I'll share it (maybe for not total, but partial). But I can't guarantee when to share. 

Thank you

Cheers

**Comment by Lisandro79** [comment]
Thanks a lot for your fast response. I did not mean to share your dataset, I was just curious on how to annotate a "background" image in COCO format.

I thought of using "FingerTip": [0, 0, 0] for the "background" images. However, I think (still need to test it) this might give me problems with the loss function during training (e.g., if a batch of images contains only images without finger tips then all images might be not considered for adjusting the gradients due to the visibility label "0"). 

I will try and see how it goes. 

Thanks a lot again
